From 12c7da459e13d37f9b02eb879f5d1c7e4dc6f287 Mon Sep 17 00:00:00 2001
From: mingwax <mingwax@codeaurora.org>
Date: Mon, 5 Jun 2017 08:54:15 +0800
Subject: [PATCH 201/211] SnapdragonCamera: SetParameters use the mParameters
 Object

User should use the mParameters take the place of mCameraDevice.
getParameters. In order to update parameter in time.

CRs-Fixed: 2054913
Change-Id: I605f05322a2a6072c40014f7ab472802c0350107
---
 src/com/android/camera/VideoModule.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/camera/VideoModule.java b/src/com/android/camera/VideoModule.java
index 81d3da8bc..b6a0184d9 100644
--- a/src/com/android/camera/VideoModule.java
+++ b/src/com/android/camera/VideoModule.java
@@ -3245,7 +3245,7 @@ public class VideoModule implements CameraModule,
             return;
         }
         forceFlashOffIfSupported(forceOff);
-        mCameraDevice.setParameters(mCameraDevice.getParameters());
+        mCameraDevice.setParameters(mParameters);
     }
 
     @Override
-- 
2.17.1

